docs/user-guide/adding-widgets.md

```diff
@@ -33,36 +33,41 @@ In dashboards, selecting the Map type widget, the following panel appears:
+* Move forward to the next step through the <img src="../img/button/next.jpg" class="ms-docbutton"/> button
 
-The next panel is similar to the following (in this case an empty map was selected):
+Once a map has been selected, a panel opens and the layers present in the map are displayed in the preview and listed in the layer list:
 
-<img src="../img/adding-widgets/wid-map-options.jpg" class="ms-docimage"  style="max-width:400px;"/>
+<img src="../img/adding-widgets/wid-layers-list.jpg" class="ms-docimage"  style="max-width:400px;"/>
 
-Now the user is allowed to add layers to the map through the <img src="../img/button/+++.jpg" class="ms-docbutton"/> button:
+!!!note
+    If the **Empty Map** is selected the user can:
 
-<img src="../img/adding-widgets/wid-add-layer.gif" class="ms-docimage"  style="max-width:400px;"/>
+    * Create a widget map using an empty map
 
-Once a layer is added to the map widget, it will be displayed in preview and in layers list:
+    * Add layers to the map through the <img src="../img/button/+++.jpg" class="ms-docbutton"/> button, as follows: <img src="../img/adding-widgets/wid-add-layer.gif" class="ms-docimage"  style="max-width:400px;"/>
 
-<img src="../img/adding-widgets/wid-layers-list.jpg" class="ms-docimage"  style="max-width:400px;"/>
+On the **Configure map options** panel the user can toggle the layer visibility and set layers transparency, as explained in the [Display options](toc.md#display-options-in-panel) section. Furthermore the user can manage the layer with the new buttons present on the layer toolbar by selecting the layer on the layers list.
 
-It's now possible to toggle the layer visibility, and set layers transparency (more information in [Display options](toc.md#display-options-in-panel) section). Furthermore, by selecting it, new buttons are added to the toolbar allowing to:
+<img src="../img/adding-widgets/wid-layers-new-buttons.jpg" class="ms-docimage"  style="max-width:400px;"/>
 
-* Zoom to layers <img src="../img/button/zoom-layer.jpg" class="ms-docbutton"/>
+Here, the user is allowed to:
 
-* Access [Layer Settings](layer-settings.md) <img src="../img/button/properties.jpg" class="ms-docbutton"/>
+* **Zoom** to layers though the <img src="../img/button/zoom-layer.jpg" class="ms-docbutton"/> button
 
-* Remove layers <img src="../img/button/delete.jpg" class="ms-docbutton"/>
+* Access [Layer Settings](layer-settings.md) though the <img src="../img/button/properties.jpg" class="ms-docbutton"/> button
 
-!!!note
-    Adding layers is not mandatory, it is possible to create a widget map using an empty map.
+* **Remove** layers though the <img src="../img/button/delete.jpg" class="ms-docbutton"/> button
+
+* Disable/Enable the [Floating Identify Tool](side-bar.md#floating-identify-tool) to retrieve information about layers on the map though the <img src="../img/button/identify_green_burron.jpg" class="ms-docbutton"/> button 
```

Comment:
* Disable/Enable the [Floating Identify Tool](side-bar.md#floating-identify-tool) to retrieve Identify information about layers available on the map through the <img src="../img/button/identify_green_burron.jpg" class="ms-docbutton"/> button